net/qpid-proton/Portfile

```diff
@@ -19,14 +18,16 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton 0.29.0
+github.setup        apache qpid-proton 0.31.0
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
@mf2k ack, will do - thanks!

Reply:
@mf2k done, let me know if there is anything else todo!